15-7-1_and_15-c-1/compile_tables.R

```diff
@@ -0,0 +1,49 @@
+# This file is directly called by update_indicator_main.R.
+# It has to be called compile_tables.R
+# It is the control script that runs all the others.
+
+library('openxlsx')
+library('stringr')
+library('janitor')
+library('tidyr')
+library('dplyr')
+
+library(SDGupdater)
+
```

Comment:
Add an if statement here (but also note that the files should just be named config and example_config):
`if (test_run == TRUE) {
source("15-7-1_15-c-1_example_config.R")
} else if (test_run == FALSE) {
  source("15-7-1_15-c-1_config.R")
} else {
  stop("test_run must be either TRUE or FALSE")
}`

Reply:
Done